robots/FrankaEmikaPandaRobot.m

```diff
@@ -0,0 +1,55 @@
+% panda = FrankaEmikaPandaRobot.kinematics() returns a DQ_kinematics object
+% using the modified Denavit-Hartenberg parameters of the Franka Emika
+% Panda robot
+
+% (C) Copyright 2015-2022 DQ Robotics Developers
+% 
+% This file is part of DQ Robotics.
+% 
+%     DQ Robotics is free software: you can redistribute it and/or modify
+%     it under the terms of the GNU Lesser General Public License as published by
+%     the Free Software Foundation, either version 3 of the License, or
+%     (at your option) any later version.
+% 
+%     DQ Robotics is distributed in the hope that it will be useful,
+%     but WITHOUT ANY WARRANTY; without even the implied warranty of
+%     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+%     GNU Lesser General Public License for more details.
+% 
+%     You should have received a copy of the GNU Lesser General Public License
+%     along with DQ Robotics.  If not, see <http://www.gnu.org/licenses/>.
+%
+% DQ Robotics website: dqrobotics.github.io
+%
+% Contributors to this file:
+%     Frederico Fernandes Afonso Silva - frederico.silva@ieee.org
+
+classdef FrankaEmikaPandaRobot
+    methods (Static)
+        function ret = kinematics()
+            % Modified D-H of Franka Emika Panda
+            FEp_DH_theta = [0, 0, 0, 0, 0, 0, 0];
```

Comment:
Let's make things simple. Change FEp_DH_theta to mDH_theta. Please, do the same for all other variables.

Reply:
Fixed in commit [8ce7d82](https://github.com/ffasilva/matlab/commit/8ce7d82d06244e07cbd5012fd2623e4b32973a3d).